Backend/src/ITL.Application/ITL.Application.csproj

```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AutoMapper" Version="12.0.1" />
+	  <PackageReference Include="AutoMapper" Version="13.0.1" />
```

Comment:
What about fixing the indentation in this line?

Reply:
Already fixed